rpm/sailfish-secrets.spec

```diff
@@ -45,10 +45,12 @@ Summary:    Unit tests for the Sailfish OS Secrets Library
 Group:      System/Libraries
 BuildRequires:  pkgconfig(Qt5QuickTest)
 BuildRequires:  pkgconfig(libcrypto)
-Requires: qt5-qtdeclarative-import-qttest
-Requires: qt5-qtdeclarative-devel-tools
-Requires: openssl
+BuildRequires:  pkgconfig(nemodevicelock)
+Requires:   qt5-qtdeclarative-import-qttest
+Requires:   qt5-qtdeclarative-devel-tools
 Requires:   libsailfishsecrets = %{version}-%{release}
+Requires:   openssl
```

Comment:
Is openssl used by the auth plugin?

Reply:
it's used by the test openssl plugin, which is also part of this test package.